lib/matplotlib/cbook/__init__.py

```diff
@@ -2156,3 +2156,65 @@ def _check_in_list(values, **kwargs):
             raise ValueError(
                 "{!r} is not a valid value for {}; supported values are {}"
                 .format(v, k, ', '.join(map(repr, values))))
+
+
+_valid_compass = ['NE', 'NW', 'SW', 'SE', 'E', 'W', 'E', 'S', 'N', 'C']
+
+
+def _map_loc_to_compass(loc, allowtuple=False, allowbest=False,
```

Comment:
it's a private api so it doesn't really matter, but these look like good parameters to make keyword only